src/hb-aat-layout-feat-table.hh

```diff
@@ -45,15 +45,25 @@ struct SettingName
     return_trace (likely (c->check_struct (this)));
   }
 
-  protected:
   HBUINT16	setting;	/* The setting. */
   NameID	nameIndex;	/* The name table index for the setting's name. */
   public:
   DEFINE_SIZE_STATIC (4);
 };
 
+struct feat;
+
 struct FeatureName
 {
+  static int cmp (const void *key_, const void *entry_)
+  {
+    hb_aat_feature_type_t key = * (hb_aat_feature_type_t *) key_;
+    const FeatureName * entry = (const FeatureName *) entry_;
+    return key < entry->feature ? -1 :
+	   key > entry->feature ? 1 :
```

Comment:
Nit, but I think we use `+1` in these cases.

Reply:
Fixed